components/Chart/Pie/PieChart.vue

```diff
@@ -8,7 +8,7 @@ import chart from '../chart.mixin'
 export default {
   name: 'PieChart',
   mixins: [chart],
-  props: { innerSize: { type: String } },
+  props: { innerSize: { type: String }, animation: { type: Boolean, default: false }, tooltipDescription: { type: String, default: undefined } },
```

Comment:
could you put that with new line like in the other components, that will be easier to read

Reply:
I don't think `innerSize` is used

Reply:
> I don't think `innerSize` is used

`innerSize` is used to make thin or thick for a pie chart. I think it needs to declare `innerSize` for custom a pie chart and set a default value to a pie chart

Reply:
> could you put that with new line like in the other components, that will be easier to read

Fixed.